openedx/core/djangoapps/call_stack_manager/core.py

```diff
@@ -23,122 +27,221 @@
 3. For tracking Save and Delete events-
     Use mixin called "CallStackMixin"
     For ex.
-        class StudentModule(CallStackMixin, models.Model):
-4. Decorator is a parameterized decorator with class name/s as argument
-    How to use -
-    1. Import following
-        import from openedx.core.djangoapps.call_stack_manager import donottrack
+        class StudentModule(models.Model, CallStackMixin):
+
+TRACKING FUNCTIONS, and METHODS-
+1. Import following-
+    from openedx.core.djangoapps.call_stack_manager import trackit
+NOTE - @trackit is non-parameterized decorator.
+
+FOR DISABLING TRACKING-
+1. Import following at appropriate location-
+    from openedx.core.djangoapps.call_stack_manager import donottrack
+** NOTE ** @donottrack accepts classes. However, while dealing with functions/methods, you should pass a string.
+ The string has a specific format i.e. '<entity_name>.__module__ + "." + <entity_name>.__name__'
```

Comment:
Why a string? Why not pass the function/method directly, just like we're passing the classes directly?